app/Metadata/Extractor.php

```diff
@@ -237,7 +272,7 @@ public function extract(string $filename, string $type): array
 			$metadata['position'] = implode(', ', $fields);
 		}
 
-		if (strpos($type, 'video') !== 0) {
```

Comment:
are you sure about that one ?

Reply:
See above...

Reply:
yeap I'm reading I'm reading. :) I just quickly commented. :p